package.json

```diff
@@ -37,7 +38,8 @@
     "objection": "2.x.x",
     "coveralls": "3.x.x",
     "@hapi/code": "8.x.x",
-    "@hapi/lab": "24.x.x"
+    "@hapi/lab": "24.x.x",
+    "@hapipal/types-avocat": ">=0"
```

Comment:
We may want to tweak this to be more similar to the others, but this is alright for testing for now!  E.g. https://github.com/hapipal/confidence/pull/117/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R24